projects/packages/forms/src/dashboard/components/layout/style.scss

```diff
@@ -37,6 +37,10 @@ body.toplevel_page_jetpack-forms {
 		align-items: center;
 		justify-content: space-between;
 		padding: 20px 150px 0 0;
+
+		@media (max-width: 660px) {
```

Comment:
I think this value should be more like 782px since that is when WP switches to mobile view.

Reply:
There are some [re-usable mixings](https://github.com/WordPress/gutenberg/blob/0fa123bb4648f8dd25adf20871a2c60ce531a0cb/packages/base-styles/_mixins.scss#L72-L128) or [breakpoint variables](https://github.com/WordPress/gutenberg/blob/0fa123bb4648f8dd25adf20871a2c60ce531a0cb/packages/base-styles/_breakpoints.scss#L5-L14). 

They're [exported](https://github.com/Automattic/jetpack/blob/6e79dddaece6bbe98f5fb06510f52f435f953300/projects/js-packages/base-styles/gutenberg-base-styles.scss#L1-L6) via [Jetpack base-styles](https://github.com/Automattic/jetpack/tree/6e79dddaece6bbe98f5fb06510f52f435f953300/projects/js-packages/base-styles#readme) or could be imported directly via `@wordpress/base-styles` as well.